helm/druid/templates/historical/roleBinding.yaml

```diff
@@ -0,0 +1,20 @@
+{{- if and (.Values.rbac.create) (.Values.historical.enabled) }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: {{ template "druid.historical.fullname" . }}
+  labels:
+    app: {{ include "druid.name" . }}
+    chart: {{ include "druid.chart" . }}
+    component: {{ .Values.historical.name }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: {{ template "druid.historical.fullname" . }}
+subjects:
+  - kind: ServiceAccount
+    name: {{ .Values.historical.serviceAccount.name }}
```

Comment:
shouldn't this be `druid.historical.serviceAccountName`? The default value of this property is empty. is the chart generating the k8s configs correctly?

Reply:
Thank you for catching this.  This was a left-over from before @dampcake recommended switching to the empty default value above in this PR and my testing had already had the `.Values.*.serviceAccount.name` values set from previous as well so I wasn't getting an issue here.

Fixed now.